torchrl/trainers/helpers/models.py

```diff
@@ -414,6 +410,295 @@ def make_ddpg_actor(
+        )
+
+        policy_net = MLP(
+            num_cells=[64],
+            out_features=out_features,
+        )
+        if not cfg.gSDE:
+            actor_net = NormalParamWrapper(
+                policy_net, scale_mapping=f"biased_softplus_{cfg.default_policy_scale}"
+            )
+            in_keys = ["hidden"]
+            actor_module = TensorDictModule(
+                actor_net, in_keys=in_keys, out_keys=["loc", "scale"]
+            )
+        else:
+            in_keys = ["hidden"]
+            gSDE_state_key = "hidden"
+            actor_module = TensorDictModule(
+                policy_net,
+                in_keys=in_keys,
+                out_keys=["action"],  # will be overwritten
+            )
+
+            if action_spec.domain == "continuous":
+                min = action_spec.space.minimum
+                max = action_spec.space.maximum
+                transform = SafeTanhTransform()
+                if (min != -1).any() or (max != 1).any():
+                    transform = d.ComposeTransform(
+                        transform,
+                        d.AffineTransform(loc=(max + min) / 2, scale=(max - min) / 2),
+                    )
+            else:
+                raise RuntimeError("cannot use gSDE with discrete actions")
+
+            actor_module = TensorDictSequential(
+                actor_module,
+                TensorDictModule(
+                    LazygSDEModule(transform=transform),
+                    in_keys=["action", gSDE_state_key, "_eps_gSD"],
+                    out_keys=["loc", "scale", "action", "_eps_gSDE"],
+                ),
+            )
+
+        policy_operator = ProbabilisticActor(
+            spec=CompositeSpec(action=action_spec),
+            module=actor_module,
+            dist_in_keys=["loc", "scale"],
+            default_interaction_mode="random",
+            distribution_class=policy_distribution_class,
+            distribution_kwargs=policy_distribution_kwargs,
+            return_log_prob=True,
+        )
+        value_net = MLP(
+            num_cells=[64],
+            out_features=1,
+        )
+        value_operator = ValueOperator(value_net, in_keys=["hidden"])
+        actor_value = ActorValueOperator(
+            common_operator=common_operator,
+            policy_operator=policy_operator,
+            value_operator=value_operator,
+        ).to(device)
+    else:
+        if cfg.from_pixels:
+            raise RuntimeError(
+                "A2C learnt from pixels require the shared_mapping to be set to True."
+            )
+        if cfg.lstm:
+            policy_net = LSTMNet(
+                out_features=out_features,
+                lstm_kwargs={"input_size": 64, "hidden_size": 64},
+                mlp_kwargs={"num_cells": [64, 64], "out_features": 64},
+            )
+            in_keys_actor += ["hidden0", "hidden1"]
+            out_keys += ["hidden0", "hidden1", "next_hidden0", "next_hidden1"]
```

Comment:
I think the last two keys should be `("next", "hidden0")` and `("next", "hidden1")` instead